gymnasium/envs/registration.py

```diff
@@ -97,27 +117,156 @@ class EnvSpec:
     disable_env_checker: bool = field(default=False)
     apply_api_compatibility: bool = field(default=False)
 
+    # Environment arguments
+    kwargs: dict = field(default_factory=dict)
+
     # post-init attributes
     namespace: str | None = field(init=False)
     name: str = field(init=False)
     version: int | None = field(init=False)
 
-    # Environment arguments
-    kwargs: dict = field(default_factory=dict)
+    # applied wrappers
+    applied_wrappers: tuple[WrapperSpec, ...] = field(init=False, default_factory=tuple)
 
-    # Vectorized environment
-    vector_entry_point: str | None = field(default=None)
+    # Vectorized environment entry point
+    vector_entry_point: VectorEnvCreator | str | None = field(default=None)
 
     def __post_init__(self):
-        """Calls after the spec is created to extract the namespace, name and version from the id."""
-        # Initialize namespace, name, version
+        """Calls after the spec is created to extract the namespace, name and version from the environment id."""
         self.namespace, self.name, self.version = parse_env_id(self.id)
 
     def make(self, **kwargs: Any) -> Env:
         """Calls ``make`` using the environment spec and any keyword arguments."""
-        # For compatibility purposes
         return make(self, **kwargs)
 
+    def make_vec(self, **kwargs: Any) -> VectorEnv:
```

Comment:
This shouldn't be added imo. The existence of `spec.make` in the first place is a backwards compatibility thing for the old, extremely spaghettified code, so we should think about deprecating and removing it, instead of extending this pattern.

Reply:
OK, agreed. Do we want to add a deprecation warning to `EnvSpec.make()`?